beacon-chain/entrypoint.sh

```diff
@@ -3,6 +3,37 @@
 # Concatenate EXTRA_OPTS string
 [[ -n "$CHECKPOINT_SYNC_URL" ]] && EXTRA_OPTS="${EXTRA_OPTS} --checkpoint-sync-url=${CHECKPOINT_SYNC_URL}"
 
+case $_DAPPNODE_GLOBAL_EXECUTION_CLIENT_MAINNET in
+"geth.dnp.dappnode.eth")
+    HTTP_ENGINE="http://geth.dappnode:8551"
+    ;;
+"nethermind.public.dappnode.eth")
+    HTTP_ENGINE="http://nethermind.dappnode:8551"
+    ;;
+"erigon.dnp.dappnode.eth")
+    HTTP_ENGINE="http://erigon.dappnode:8551"
+    ;;
+"besu.dnp.dappnode.eth")
```

Comment:
```suggestion
"besu.public.dappnode.eth")
```
Nabsku ended up publishing the public version which I think is missing s couple important commits at least for aesthetic purposes etc. For mainnet we have the goerli under our control published as s DNP.